ocs_ci/deployment/aws.py

```diff
@@ -631,7 +631,7 @@ def remove_rhcos_workers(self):
             cordon = f"oc adm cordon {node}"
             run_cmd(cordon)
             drain = (
-                f"oc adm drain {node} --force --delete-local-data "
+                f"oc adm drain {node} --force --delete-emptydir-data "
```

Comment:
`--delete-emptydir-data` From which version is it deprecated and from which version of oc is this supported?

If there are some versions then we should make this based on some condition I guess.